gdk-sys/src/lib.rs

```diff
@@ -626,4 +630,35 @@ extern "C" {
     // Gdk Key Handling                                                  NOT OK
     //=========================================================================
     pub fn gdk_keyval_name                         (keyval:c_uint) -> *mut c_char;
+
+    //=========================================================================
+    // GdkPixbufLoader                                                   NOT OK
```

Comment:
It means so it hasn't been tested yet. A big part of rust-gnome is in the same case.